apps/user-office-backend/src/eventHandlers/messageBroker.ts

```diff
@@ -287,23 +282,49 @@ export function createPostToRabbitMQHandler() {
           };
         });
 
-        const json = JSON.stringify(answers);
+        const jsonMessage = JSON.stringify(answers);
 
-        await rabbitMQ.sendMessageToExchange(EXCHANGE_NAME, event.type, json);
+        await rabbitMQ.sendMessageToExchange(
+          EXCHANGE_NAME,
+          event.type,
+          jsonMessage
+        );
         break;
       }
       case Event.CALL_CREATED: {
-        const json = JSON.stringify(event.call);
+        const jsonMessage = JSON.stringify(event.call);
 
-        await rabbitMQ.sendMessageToExchange(EXCHANGE_NAME, event.type, json);
+        await rabbitMQ.sendMessageToExchange(
+          EXCHANGE_NAME,
+          event.type,
+          jsonMessage
+        );
         break;
       }
     }
   };
 }
 
-export function createListenToRabbitMQHandler() {
-  const rabbitMQ = getRabbitMQMessageBroker();
+export async function createListenToRabbitMQHandler() {
+  const EVENT_SCHEDULING_QUEUE_NAME = process.env
+    .EVENT_SCHEDULING_QUEUE_NAME as Queue;
+  const SCHEDULER_EXCHANGE_NAME = process.env.SCHEDULER_EXCHANGE_NAME;
+
+  // TODO: Maybe we need to check if scheduler is enabled then throw these errors. Check this with STFC.
```

Comment:
Yeah, could this only be done if the scheduler feature flag is set?

Reply:
I will try to do that

Reply:
@ACLay do you want to completely disable the `createListenToRabbitMQHandler` and `createPostToRabbitMQHandler` if the scheduler is disabled or just avoid throwing the error messages? Even if we avoid throwing the errors it will still try to listen on a queue that has empty value because that `env` variable is not set.

Reply:
@TCMeldrum did a bit more checking on this. Looks like we can just avoid using this listener altogether by updating the STFC [dependencyConfig file](https://github.com/UserOfficeProject/user-office-core/blob/235c0b687b3bbbee6044708bcd01d191d980f28d/apps/user-office-backend/src/config/dependencyConfigSTFC.ts#L79) so it uses a dummy listener like in the [test depencyConfig](https://github.com/UserOfficeProject/user-office-core/blob/235c0b687b3bbbee6044708bcd01d191d980f28d/apps/user-office-backend/src/config/dependencyConfigTest.ts#L75). If that is changed it should prevent this causing errors on our side, and this TODO comment can be removed.

Reply:
@ACLay thanks for letting me know. I will try to update that and you can review if it is good enough.

Reply:
@ACLay just to be clear that I will only skip the listener but not the `createPostToRabbitMQHandler`.